build/common.props

```diff
@@ -55,7 +55,6 @@
   </ItemDefinitionGroup>
 
   <ItemGroup>
-    <MetaPackagePackageReference Include="Microsoft.AspNetCore.Certificates.Configuration" />
```

Comment:
We like to keep the package references in the same file for this repo. Create another item group MetaPackageSourceReference?